commitlint.config.js

```diff
@@ -1 +1,6 @@
-module.exports = {extends: ['@commitlint/config-conventional']}
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'scope-enum': [2, 'always', ['android', 'ios']],
```

Comment:
Will this allow commits without scope?

Reply:
Yes, this allows default types without scopes and also these specific two scope.